src/transformStateWithClones.js

```diff
@@ -66,6 +66,30 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const result = [];
+  const clone = { ...state };
+
+  for (const transform of transforms) {
+    switch (transform.operation) {
+      case 'addProperties':
+        Object.assign(clone, transform.properties);
+        break;
+
+      case 'removeProperties':
+        for (const property in transform.properties) {
+          delete clone[transform.properties[property]];
+        };
+        break;
+
+      case 'clear':
+        for (const key in clone) {
+          delete clone[key];
+        }
```

Comment:
```suggestion
      case 'clear':
        for (const key in clone) {
          delete clone[key];
        }
        break;
```
Лучше ставить break; для каждого case.